lib/reports/ppl-sla/index.js

```diff
@@ -8,58 +8,91 @@ module.exports = ({ db, query: params, flow }) => {
       .leftJoin('activity_log', 'cases.id', 'activity_log.case_id')
       .select([
         'cases.*',
-        db.flow.raw('JSON_AGG(activity_log.*) as activity')
+        db.flow.raw('JSON_AGG(activity_log.* ORDER BY activity_log.created_at asc) as activity')
       ])
-      .whereRaw(`data->>'model' = 'project'`)
-      .whereRaw(`data->>'action' = 'grant'`)
-      .whereRaw(`data->'modelData'->>'status' = 'inactive'`)
+      .whereRaw(`cases.data->>'model' = 'project'`)
+      .whereRaw(`cases.data->>'action' = 'grant'`)
+      .whereRaw(`cases.data->'modelData'->>'status' = 'inactive'`)
       .where('cases.created_at', '<', moment().subtract(40, 'days').format('YYYY-MM-DD'))
+      .where(function () {
+        // ignore comment-related activity
+        this
+          .where('activity_log.event_name', 'like', 'status:%')
+          .orWhere('activity_log.event_name', '=', 'update');
```

Comment:
Should improve performance a decent amount by filtering to only the activity we need.